src/components/ha-climate-control.html

```diff
@@ -67,38 +70,47 @@
       },
     };
   }
+  temperatureStateInFlux(inFlux) {
+    this.$.target_temperature.classList.toggle('in-flux', inFlux);
+  }
   incrementValue() {
-    var newval = this.value + this.step;
-    this.last_changed = Date.now();
+    const newval = this.value + this.step;
     if (newval <= this.max) {
-      this.value = newval;
+      // If no initial target_temp
+      // this forces control to start
+      // from the min configured instead of 0
+      if (newval <= this.min) {
+        this.value = this.min;
```

Comment:
I think
```
this.last_changed = Date.now();
this.temperatureStateInFlux(true);
```
should apply to this case too.

Reply:
I improved the detection of value changed to put temperature color correctly and trigger last_changed trigger